addons/mail/static/src/components/composer/composer.scss

```diff
@@ -75,7 +75,8 @@
     grid-area: core-header;
 }
 
-.o_Composer_cancelReply {
+.o_Composer_cancelReply,
+.o_Composer_replyingToMessage{
```

Comment:
Avoid `,` in selectors this is a bit confusing and hard to maintain. Just duplicate this rule if necessary.